Softeq.XToolkit.Bindings.Tests/MockBinding.cs

```diff
@@ -0,0 +1,43 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Linq.Expressions;
+
+namespace Softeq.XToolkit.Bindings.Tests
+{
+    public class MockBinding<TSource, TTarget> : Binding<TSource, TTarget>
```

Comment:
We might think of introducing an IBinding for our Bindings. It shouldn't really affect usage, but it will reduce coupling and will make unit-testing much easier.
And with the current implementation, it might be useful that NSubstitute actually allow to instantiate abstract classes and define logic of abstract members using same Sbstitute.For<>() API

Reply:
You right, a good place for `Substitute.For<>()`!
Fixed